<launch>

  <node pkg="rosserial_python" name="rosserial_braccio" type="serial_node.py" output="screen" clear_params="true">
    <param name="port" value="/dev/ttyACM0" />
  </node>

   <node pkg="su_chef" name="parse_and_publish_braccio" type="parse_and_publish" output="screen">
  </node>

  <node pkg="rosserial_python" name="rosserial_slicer" type="serial_node.py" output="screen" clear_params="true">
    <param name="port" value="/dev/ttyACM1" />
  </node>

</launch>
